README.md

```diff
@@ -67,8 +67,8 @@ If you see a url response, you are ready to go.
 
 #### Deploy Rook
 
-Rook can be setup and deployed in Kubernetes by simply deploying the [rook-operator](https://github.com/rook/rook/blob/master/demo/kubernetes/rook-operator.yaml) deployment manifest.
-You will this manifest and all our example manifests files in the [demo/kubernetes](https://github.com/rook/rook/blob/master/demo/kubernetes) folder.
+Rook can be setup and deployed in Kubernetes by simply deploying the [rook-operator](demo/kubernetes/rook-operator.yaml) deployment manifest.
+You will this manifest and all our example manifests files in the [demo/kubernetes](demo/kubernetes) folder.
```

Comment:
Typo: `You will this manifest` ("You will _find_ this manifest")
Typo: `all our example manifests files` (`s/manifests/manifest` OR `s/files //`)